show/reboot_cause.py

```diff
@@ -24,6 +23,100 @@ def read_reboot_cause_file():
     return reboot_cause_dict
 
 
+# Function to fetch reboot cause data from database
+def fetch_data_from_db(module_name, fetch_history=False, use_chassis_db=False):
```

Comment:
@rameshraghupathy how are these functions prevented from executing on non-smartswitch platforms?

Reply:
The KEY has unique identifier "DPUx" which will prevent the other platforms.
@prgeor